src/scss/style.scss

```diff
@@ -8,28 +8,421 @@
 }
 
 .lr-zoom-in-icon {
-  background-image: url("../assets/images/icons/zoom-in.svg");
+  background-image: url("~@/assets/images/icons/zoom-in.svg");
 }
 .lr-zoom-in-icon:hover {
-  background-image: url("../assets/images/icons/zoom-in-hover.svg");
+  background-image: url("~@/assets/images/icons/zoom-in-hover.svg");
 }
 
 .lr-tooltip-text {
-  max-width:100px;
-  padding:5px;
-  min-height:10px;
-  background:#000;
+  max-width: 100px;
+  padding: 5px;
+  min-height: 10px;
+  background: #000;
   visibility: hidden;
   color: #fff;
   text-align: center;
   border-radius: 6px;
   /* Position the tooltip */
   position: absolute;
   z-index: 1;
-  word-wrap:break-word;
+  word-wrap: break-word;
 }
 .lr-tooltip:hover {
   .lr-tooltip-text {
-  visibility: visible;
+    visibility: visible;
+  }
 }
+
+/* Textbox style rules --- Start */
+.sgt-form-group {
```

Comment:
If you use vuetify buttons you don't need these class

Reply:
Removed the unused styles.